scylla-server/src/services/systems.services.ts

```diff
@@ -1,11 +1,29 @@
 import prisma from '../prisma/prisma-client';
-import { ResponseFunction } from '../utils/message-maps.utils';
 
 /**
  * CRUD operation to get all systems with ResponseFunction type
- * @returns string contianing all the systems in the db
+ * @returns Promise<string> contianing all the systems in the db
  */
-export const getAllSystems: ResponseFunction = async () => {
+export async function getAllSystems(): Promise<string> {
```

Comment:
No id keep it the way it was

Reply:
with the current response function typing, getAllSystems is required to take in data, which I didn't like because the function just gets all of the systems it doesn't require any input. i can revert it back tho no prob

Reply:
It needs to be of type response function which is the thing, you could alter ResponseFunction to be have data be optional